graphene/contrib/django/filter/fields.py

```diff
@@ -26,7 +27,13 @@ def get_queryset(self, qs, args, info):
         filter_kwargs = self.get_filter_kwargs(args)
         order = self.get_order(args)
         if order:
-            qs = qs.order_by(order)
+            qs = qs.order_by(to_snake_case(order))
```

Comment:
Other question: what if `order` is a list? https://docs.djangoproject.com/en/1.10/ref/models/querysets/#django.db.models.query.QuerySet.order_by

To do multi field ordering (in your test `order_by('first_name', 'articles')`)

`django_filters.filterset.BaseFilterSet`
```                    qs = qs.order_by(*self.get_order_by(ordered_value))
```
would handle it, though (since I do not need a nice patch from you for this one) it would not handle the nice `to_snake_case` you added.